vpr/src/route/rr_graph.xsd

```diff
@@ -234,12 +234,59 @@
     </xs:restriction>
   </xs:simpleType>
 
+  <!-- The enumeration MUST follow the order of side maskcode
```

Comment:
Do not rely on the enumeration order matching the mask.  Preserve the previous order, and use a `std::array<>` array map to translate from the enum to the bitmask.

Reply:
Not a problem for me. I was trying to compress the codes. I use `if-else` tree to translate.